cypress/utils/apps/data_explorer/commands.js

```diff
@@ -162,3 +162,187 @@ function checkForElementVisibility() {
       }
     });
 }
+
```

Comment:
- All the newly added functions here are regular functions, not cypress commands. Since this file is `commands.js`, only Cypress.Commands.add functions should be here
- So we got two options to rectify this. Either: 1) Make all these functions into cypress custom commands or 2) move these functions out to another file . I am going to vote for choice 2 for the following reason:
    - [Cypress best practices recommends it](https://docs.cypress.io/api/cypress-api/custom-commands#Best-Practices) (See `1. Don't make everything a custom command`) (I personally feel that we use too much cypress commands in the codebase already)
    
As to where we should move these... This is just a suggestion but what I did was create a `utils` directory next to the tests itself: https://github.com/opensearch-project/OpenSearch-Dashboards/pull/9112/files#diff-80ee042bf13b443df19ca0fbb20fc6242ca982791beef74d65eb1d8d74f39f89